### PR TITLE
chore: Inherit stdio

### PR DESCRIPTION
**Pull request by dnys1**
Inherits stdio in `amplify init` and `amplify push` so that those commands' logs are streamed to the gen process.

### CODE REVIEW

File: hooks/post_gen.dart [hunk 1]
```diff
@@ -55,16 +55,18 @@ Future<void> _runAmplifyInit(HookContext context) async {
...
   final exitCode = await result.exitCode;
 
   if (exitCode == 0) {
-    amplifyInitProgress.complete('Project is initialized.');
+    context.logger.success('Project is initialized.');
   } else {
-    final errorBytes = await result.stderr.first;
-    final error = systemEncoding.decode(errorBytes);
-    amplifyInitProgress.complete('Project initialization failed. $error');
+    context.logger.err('Project initialization failed.');
```
Comment (salihgueler) [inline hooks/post_gen.dart:69]: Out of curiosity: Will this share the error message as well?

**Review by salihgueler — APPROVED** [review]
Thanks for the contribution!

File: hooks/post_gen.dart (hunk 1 above)
Comment (dnys1) [reply hooks/post_gen.dart:69]: Yep! The child process will inherit both stdout and stderr. Here's how a failure looks

Screen Shot 2022-08-04 at 4 25 34 PM

Comment (salihgueler) [reply hooks/post_gen.dart:69]: This is awesome! Thank you!

**Pull request merged**